README.md

```diff
@@ -42,15 +42,21 @@ The intended usage is as an npm script:
 }
 ```
 
+Then run it with: `$ npm run -s eslint-find-option-rules` (the `-s` is to silence npm output).
```

Comment:
probably best to use `--silent`, rather than abbreviations - it's also more googleable.